src/store/actions/actions.type.js

```diff
@@ -1 +1,5 @@
 //This is the file to list all actionsß
+
+export const COHORTS = {
```

Comment:
I think we should have a loader or something like this and for this reason I would rename it to `ACTION_TYPES`

Reply:
Something like this
`export const ACTION_TYPES = {
  GET_COHORTS_PENDING: "sent cohorts request",
  GET_COHORTS_SUCCESS: "recieve cohorts success",
  GET_COHORTS_FAILURE: "recieve cohorts failure"
};`

This can be done in a different PR though

Reply:
@martapa So, I was thinking we might have different objects for each different set of actions/reducers. For the single cohort actions and reducers we may have a top level object called COHORT and within the COHORT and COHORTS action type objects there could be pending actions to track that they're loading. What do you think?

Reply:
I see. Is this popular approach?

Reply:
It works for me. I was just asking, because I wasn't sure what's the plan :p

Reply:
@martapa Just something I thought made sense - not a convention that I've seen. Maybe we can discuss further tomorrow :)